lib/usd/translators/CMakeLists.txt

```diff
@@ -33,18 +34,28 @@ target_sources(${TARGET_NAME}
         meshReader.cpp
         meshWriter.cpp
         meshWriter_Primvars.cpp
-        meshWriter_Skin.cpp
-        nurbsCurveWriter.cpp
         nurbsCurvesReader.cpp
+        nurbsCurveWriter.cpp
         nurbsPatchReader.cpp
         nurbsSurfaceWriter.cpp
         particleWriter.cpp
         scopeReader.cpp
-        strokeWriter.cpp
         skelReader.cpp
+        strokeWriter.cpp
         xformReader.cpp
 )
 
+set(headers
+    jointWriteUtils.h
+)
+
+# -----------------------------------------------------------------------------
+# promoted headers
+# -----------------------------------------------------------------------------
+mayaUsd_promoteHeaderList(HEADERS ${headers}
+                          BASEDIR ${TARGET_NAME}
+)
+
```

Comment:
If the intention is to move all the joint and skin related stuff into a reusable library, shouldn't this all go in `lib/mayaUsd/fileio/utils` alongside all the other read/write utils? It doesn't seem appropriate for this translators library to be the thing that provides them, especially so since it doesn't currently provide **any** headers at all.

Reply:
@mattyjams Yes, good point. I can't agree more. not really sure how I missed this.:) 

Please see 2d150798c6f9abc919325d0badf6dd4a185e3381